pages/learn/manage/actions.md

```diff
@@ -32,7 +32,15 @@ You may also enable or disable public device URLs by clicking the _Public device
 
 The `Restart Application` action restarts the currently running **application containers**. Your application's running containers will be removed and recreated from scratch. This behavior is intended, and is different from running `balena restart [OPTIONS] CONTAINER [CONTAINER...]` in a host OS terminal instance from your dashboard, which will not remove your containers. If you are trying to persist data between container removals, see [persistent storage][persistent-storage] for strategies.
 
-When the containers are stopped, the application is politely asked to stop by sending a `SIGTERM` and after 10 seconds of wait time a `SIGKILL` is sent.
+By removing containers and recreating them from scratch, we see benefits like the following:
+
+- Containers are meant to be ephemeral, meaning that a new container should be a drop-in replacement for an old container with minimal to no impact. Removing and recreating containers adheres to this philosophy.
+
+- Because containers are removed and recreated with the restart action, you're encouraged to follow best practices in Docker data persistence. For more information, see our [persistent storage][persistent-storage] documentation or [Docker's data persistence strategies][docker-data-persistence-strategies]. These strategies also offer a performance boost over storing files in the container's writable layer.
+
+- Removing and recreating containers is useful in a scenario where recreating a container allows recovery from application bugs where the application got stuck in an invalid state that was persisted to the container's filesystem. In this scenario, data loss is a good thing, as the data consists of a bad state.
```

Comment:
```suggestion
- Removing and recreating containers may allow recovery from application bugs where the application is stuck in an invalid state. For example, a process ID file that is no longer valid but is persisted to the container filesystem would be cleaned up when recreating the container.
```
I think mentioning that the "data loss is a good thing" isn't super helpful to the point. Please feel free to disagree with me!

Reply:
No disagreement here 👍🏼